### DISCUSSION TITLE
Reloadable-input field type -"text"

### DISCUSSION THREAD

**Discussion by rsaiyad** [cypress-io/cypress#23398] [Questions and Help]
I am trying to automate my test caese in Cypress. I have en input field of text i.e. customer Id. After inserting the customer Id other fields i.e. customer's name, adress, telephone etc will be field up automatically if the customer exists in the database. The customer ID that I am using exists in the database and it works fine when I insert the ID manuelly. But when I run the Cypress script it doesn't. Here is my code: 
it ('Cutsomer information' , () => {
    cy.get('#ID').click().type('1234567')
})
I have even tried with 'enter' but not working.
Is there any idea or any alternative to insert a value which will reload the other fields?

**Comment by jjhelguero** [comment]
For your automation test, do you mimic exactly what you do for manual?
Sending a click to the element seems redundant when you have a `.type()` command following it. Also, it is good to have assertions, such as `.should('be.visible')` before you send an action command. Another note, you may have to dig deeper into your app and how it checks with the database if the ID exists, then you can wait on that.

**Reply by rsaiyad** [reply]
Thanks for your answer, but I have already tested with .should('be.visible'), type('1234567',{force : true}) etc. Nothing works. I even used realClick, trigger etc. I think I have to dig the app's code and check how it behave when run the script.

**Reply by rsaiyad** [reply]
My problem is solved. I don't know why Cypress behave so. The reason was I divided my test in different test cases i.e. log in, cutomer's name, data validation etc put in separet it function. But when deleted all test cases and put dem in to one it function. It works fine. Can any one tell me the reason?
